src/api/models/mint-token-data-v2.ts

```diff
@@ -29,6 +29,7 @@ export interface MintTokenDataV2 {
     'blueprint'?: string;
     /**
      * Token ID
+     * Note: While the Token ID is required to be a string, it still needs to be a valid uint256 as per the ERC-721 token standard.
```

Comment:
Why be vague here, I think we should just say:

"Note: Token ID is a string to prevent overflow issues on large uint256 inputs. It needs to be a valid number and cannot have leading zeroes"

Also, this is typescript, can we not shadow the type somehow? It must be confusing to end users that this is a string, can we create something like `type Uint256 = string`.

As far as I understand TS this should not break existing clients because TS accepts regular strings as this type

```
type UInt256 = string;
let a: UInt256 = "asdf"
a = "another string"

function test(a: UInt256) {
    return a
}

console.log(test("some string"))
```

Not to blow this out of proportion but maybe we can use this opportunity to improve the API.

Reply:
> "Note: Token ID is a string to prevent overflow issues on large uint256 inputs. It needs to be a valid number and cannot have leading zeroes"

I see your point. My intention was to keep it as concise as we can and not leak any additional information as to _how_ we store this information. I don't mind either TBH, @immutable/imx-games-squad  any opinions?

> Also, this is typescript, can we not shadow the type somehow? It must be confusing to end users that this is a string, can we create something like type Uint256 = string.

I do see some value in this but I also think it could confuse end users as `uint256` can actually refer to a type in our context. Also it won't stop them from appending zeroes or non-numeric characters to the string so I also question the value it would provide.

And this could possibly addressed in the cleanup work if we think adding such a type adds value. Thoughts @immutable/imx-games-squad  ?